src/vs/editor/contrib/codeAction/browser/codeActionCommands.ts

```diff
@@ -490,3 +503,61 @@ export class AutoFixAction extends EditorAction {
 			CodeActionAutoApply.IfSingle, undefined, CodeActionTriggerSource.AutoFix);
 	}
 }
+
+const CodeActionContribution = EditorCommand.bindToContribution<QuickFixController>(QuickFixController.get);
+
+const weight = KeybindingWeight.EditorContrib + 90;
+
+registerEditorCommand(new CodeActionContribution({
+	id: 'hideCodeActionMenuWidget',
+	precondition: Context.Visible,
+	handler(x) {
+		x.hideCodeActionMenu();
+	},
+	kbOpts: {
+		weight: weight,
+		primary: KeyCode.Escape,
+		secondary: [KeyMod.Shift | KeyCode.Escape]
+	}
+}));
+
+registerEditorCommand(new CodeActionContribution({
+	id: 'navigatePrevious',
```

Comment:
I think these commands should all have ids that mention code actions. For example:

`selectPreviousCodeAction`
`selectNextCodeAction`